com.unity.shadergraph/Editor/Data/Nodes/Procedural/Shape/RoundedRectangleNode.cs

```diff
@@ -30,7 +30,8 @@ protected override MethodInfo GetFunctionToConvert()
     Radius = max(min(min(abs(Radius * 2), abs(Width)), abs(Height)), 1e-5);
     $precision2 uv = abs(UV * 2 - 1) - $precision2(Width, Height) + Radius;
     $precision d = length(max(0, uv)) / Radius;
-    Out = saturate((1 - d) / fwidth(d));
+    $precision fwd = max(fwidth(d), 1e-5);
```

Comment:
Smallest half (fp16) is 0.00006103... which is > 0.00001, maybe use a bit larger value here, or use HALF_MIN defined in Macros.hlsl.
Otherwise LGTM.

Reply:
I was re-using `1e-5` as it already appeared earlier in the function.